snapcraft/parts/plugins/matter_plugin.py

```diff
@@ -0,0 +1,173 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""The matter plugin."""
+import os
+
+from typing import Any, Dict, List, Set, cast
+
+from craft_parts import infos, plugins
+from overrides import overrides
+
+MATTER_REPO = "https://github.com/project-chip/connectedhomeip.git"
+"""The repository where the matter SDK resides."""
+
```

Comment:
```suggestion
# The repository where the matter SDK resides.
MATTER_REPO = "https://github.com/project-chip/connectedhomeip.git"
```

Reply:
+1 unless you want developers to see this URL and comment in the auto-generated documentation (which currently don't exist for snapcraft)

It will render [this](https://github.com/canonical/craft-parts/blob/8ecf07c0c22b607039642af956ba0b9d400b860e/craft_parts/plugins/validator.py#L32) to [this](https://canonical-craft-parts.readthedocs-hosted.com/en/latest/reference/gen/craft_parts.plugins.validator.html#craft_parts.plugins.validator.COMMAND_NOT_FOUND)

Reply:
Thanks for the explanation of snapcraft rendering. The suggestion has been applied in the latest commit